tests/CMakeLists.txt

```diff
@@ -314,6 +314,17 @@ if(BUILD_SOTA_TOOLS)
     endif(OAUTH2_TEST_CLIENT_ID)
 endif(BUILD_SOTA_TOOLS)
 
+
+add_test(NAME garage-deploy-option-help
+    COMMAND garage-deploy --help)
+
+add_test(NAME garage-deploy-nonexistent-option
+    COMMAND garage-deploy --test )
+set_tests_properties(garage-deploy-nonexistent-option PROPERTIES WILL_FAIL TRUE)
+
+add_test(NAME garage-deploy-build-deb COMMAND cpack WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
```

Comment:
I get a test failure when running this test, which I suspect is because the executables have not been built. I think that it isn't necessary to run this as a test: just making the 'qa' target depend in 'package' should be enough